common-api/src/main/java/org/pmiops/workbench/cdr/dao/DomainInfoDao.java

```diff
@@ -20,7 +20,7 @@
    */
   @Query(value="select new org.pmiops.workbench.cdr.model.DomainInfo(\n" +
       "d.domain, d.domainId, d.name, d.description,\n" +
-      "d.conceptId, 0L, COUNT(*), 0L)\n" +
+      "d.conceptId, 0L, COUNT(*), d.participantCount)\n" +
```

Comment:
Could you give some background on this change? Was this a bug? Or a functional change?

Reply:
According to this screenshot, https://projects.invisionapp.com/d/main#/console/13577480/339141960/preview
the search requires the participant counts on the cards.
The above query returned 0 as participant count before and the participant counts are not displayed on the cards when the search word was present. Counts were displayed on cards only when search word was absent.